.github/workflows/setup-env.yml

```diff
@@ -30,4 +30,15 @@ jobs:
             echo 'EOF'
           } >> $GITHUB_OUTPUT
           
-          rm ./vars.txt
\ No newline at end of file
+          rm ./vars.txt
+      - name: Configure sysctl limits
```

Comment:
I don't think this should live here. If this is going to be reused, then we can create another reusable workflow, but it seems like the only place this will be used is for e2e tests